README.md

```diff
@@ -15,12 +15,6 @@ Add this line to your application's Gemfile:
 gem 'rollbar', '~> 2.7.0'
 ```
 
-If you are not using JRuby we suggest using [Oj](https://github.com/ohler55/oj) for JSON serialization. In order to install Oj you can add this line to your Gemfile:
```

Comment:
Tweaked the language in https://github.com/rollbar/rollbar-gem/commit/18e064850df13b592d8f48972d50f94a67d4e01a ; @ootoovak is that more clear?